debian/kolibri-server.service

```diff
@@ -11,7 +11,7 @@ Type=forking
 ExecStart=/etc/init.d/kolibri-server start
 ExecStop=/etc/init.d/kolibri-server stop
 TimeoutStartSec=infinity
-TimeoutStopSec=10
+TimeoutStopSec=90
```

Comment:
We have done it in the past and the result was that rebooting the server, or just restarting this daemon took ages. If there are upgrade issues they should be deal separately, they should not affect to the way the service behaves in every ocasion.

Reply:
Hmm, yes, I just don't know if it's supposed to be waiting with good reason.. Raspberry Pi is a slow platform :)